db/main/migrate/20190109000000_create_beta_migration_request_table.rb

```diff
@@ -0,0 +1,11 @@
+class CreateBetaMigrationRequestTable < ActiveRecord::Migration[4.2]
+  def change
+    create_table :beta_migration_requests do |t|
+      t.integer  :owner_id
```

Comment:
I am wondering if it would make sense to add indices for the (composed) foreign key (`owner_id` and `owner_type`) to be able to fetch records faster from the DB later on? Wdyt?

Reply:
Sounds good, added!